### ISSUE TITLE
Could you add more detail of building ObjRenderer? 

### ISSUE THREAD

**Issue by spk921** [silverbottlep/tvsn#2]
I have install freeImageplus-dev. But was not able to build the ObjRenderer. 

Do you have same problem while building the ObjRenderer?

"make" -f nbproject/Makefile-Release.mk QMAKE= SUBPROJECTS= .build-conf
make[1]: Entering directory '/home/spk/git/ObjRenderer'
"make"  -f nbproject/Makefile-Release.mk dist/Release/GNU-Linux-x86/objrenderer
make[2]: Entering directory '/home/spk/git/ObjRenderer'
mkdir -p dist/Release/GNU-Linux-x86
g++ -std=c++0x    -o dist/Release/GNU-Linux-x86/objrenderer build/Release/GNU-Linux-x86/ObjRenderer.o build/Release/GNU-Linux-x86/ShaderData.o build/Release/GNU-Linux-x86/Viewer.o build/Release/GNU-Linux-x86/main.o build/Release/GNU-Linux-x86/makeMaterial.o `pkg-config --libs opencv` `pkg-config --libs OpenEXR`   -lGL -lglut -lGLU -lGLEW -lfreeimageplus
/usr/bin/ld: build/Release/GNU-Linux-x86/ObjRenderer.o: undefined reference to symbol 'FreeImage_ConvertTo8Bits'
//usr/lib/x86_64-linux-gnu/libfreeimage.so.3: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
nbproject/Makefile-Release.mk:66: recipe for target 'dist/Release/GNU-Linux-x86/objrenderer' failed
make[2]: *** [dist/Release/GNU-Linux-x86/objrenderer] Error 1
make[2]: Leaving directory '/home/spk/git/ObjRenderer'
nbproject/Makefile-Release.mk:63: recipe for target '.build-conf' failed
make[1]: *** [.build-conf] Error 2
make[1]: Leaving directory '/home/spk/git/ObjRenderer'
nbproject/Makefile-impl.mk:39: recipe for target '.build-impl' failed
make: *** [.build-impl] Error 2

**Comment by kyungjaelee** [comment]
I got the same error msg

g++ -std=c++0x    -o dist/Release/GNU-Linux-x86/objrenderer build/Release/GNU-Linux-x86/ObjRenderer.o build/Release/GNU-Linux-x86/ShaderData.o build/Release/GNU-Linux-x86/Viewer.o build/Release/GNU-Linux-x86/main.o build/Release/GNU-Linux-x86/makeMaterial.o `pkg-config --libs opencv` `pkg-config --libs OpenEXR`   -lGL -lglut -lGLU -lGLEW -lfreeimageplus
/usr/bin/ld: build/Release/GNU-Linux-x86/ObjRenderer.o: undefined reference to symbol 'FreeImage_ConvertTo8Bits'
//usr/lib/x86_64-linux-gnu/libfreeimage.so.3: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
nbproject/Makefile-Release.mk:66: recipe for target 'dist/Release/GNU-Linux-x86/objrenderer' failed

Does anybody know how to resolve this?

**Comment by silverbottlep** [comment]
@spk921 @kyungjaelee sorry for late response, Unfortunately, I have no idea what is going on here, I didn't write ObjRenderer code, you might want to contact original author.

**Comment by spk921** [comment]
@silverbottlep Thank you for your reply. However, how did you generate the data if you was not able to  compile ObjRenderer? Do you have no problem during compiling? @silverbottlep Since many of people encounter error of compiling it then could you upload the training set and test set? If the code is here to reproduce then the dataset should be upload with it which most of opensource do that. I think it is good for people as well to share the dataset at least to try out.

**Comment by silverbottlep** [comment]
@spk921 I had no problem to build objrenderer. I didn't put the rendered images here since they are huge. I put rendered chair images on google drive([chair](https://drive.google.com/file/d/0B-r7apOz1BHAbWwzTWhzWTJUeUk/view?usp=sharing)), could you try this?

**Comment by spk921** [comment]
@silverbottlep  Thank you for the data. What is your computer environment for compiling the ObjRenderer?

**Comment by silverbottlep** [comment]
@spk921 what do you mean by environment?, it was ubuntu 14.04

**Comment by spk921** [comment]
@silverbottlep Yes OS. I am using ubuntu 16.04.03. Thank you for the information. 
@kyungjaelee I assume that this make  the compiling issue.

**Comment by itswyri** [comment]
@silverbottlep @spk921 Hi, I'm the same guy as @kyungjaelee 
I had a trouble when I was using @kyungjaelee 's computer and I finally resolve the problem.
As I remove all codes after I finished my work, I cannot remember the exact file name that I modified in the ObjRenderer folder.
However, if you look into some codes in the ObjRenderer for compile, you can find a file that contains line with
"pkg-config --libs opencv pkg-config --libs OpenEXR -lGL -lglut -lGLU -lGLEW -lfreeimageplus" options.

At the end of the option, adding "-lfreeimage" solves the problem.

The missing option "-lfreeimage" to use libfreeimage (not libfreeimageplus) for compile yields the error msg and it seems that "-lfreeimageplus" option does not automatically link to libfreeimage.

**Comment by itswyri** [comment]
You can find it at line 67 of nbproject/Makefile-Release.mk

**Comment by silverbottlep** [comment]
@itswyri Thanks!

**Comment by ashwinpathak20** [comment]
Hi @silverbottlep , can you please share rendered car images too?
